contracts/PinningManager.sol

```diff
@@ -209,66 +197,31 @@ contract PinningManager {
             numberOfPeriodsDeposited <= MAX_UINT64,
             "PinningManager: period too long"
```

Comment:
Would it be costly to add better (eq. longer) error message? Because this error won't tell user much. Especially he will think that he chose wrong period and might not realize that `period * numberOfPeriodsDeposited` exceed the `StorageOffer.maximumDuration`

Reply:
It is best practice to make the error message in Solidity very brief, as it is indeed costly (when deploying the contract and when emitting the error) to have a long error message. The client can wrap this error and provide a better reason. However, if you have a suggestion to make the error message better without increasing the size too much, I'm all ears.

Reply:
Oh ok, good to know. I guess we can keep it like this then. Thanks.